src/olympia/reviewers/models.py

```diff
@@ -80,7 +80,13 @@ class CannedResponse(ModelBase):
     response = models.TextField()
     sort_group = models.CharField(max_length=255)
```

Comment:
`sort_group` and `category` fulfil quite similar needs... no?  Do we need both?

Reply:
They do, `sort_group` is very specific to how the reviewer form works though and I'd replace `sort_group` with `category` eventually ideally. At least, that was my thought, that way we can use both tools in parallel.

Additionally `sort_group` only allows entries specific to actual reviewer-actions (`reject`, `public`, `prelim` etc) so they don't allow free information to be added.